hkmc2/jvm/src/test/scala/hkmc2/JSBackendDiffMaker.scala

```diff
@@ -42,7 +48,7 @@ abstract class JSBackendDiffMaker extends MLsDiffMaker:
     if js.isSet then
       val low = ltl.givenIn:
         codegen.Lowering()
-      val jsb = codegen.js.JSBuilder()
+      val jsb = if jsSanityCheck.isSet then JSBuilderSanityCheck() else JSBuilder()
```

Comment:
Re suggestions above:
```suggestion
      val jsb = new JSBuilder with JSBuilderSanityCheck(noSanityChecks.isUnset)
```